test/ctia/bundle/core_asset_ref_entities_test.clj

```diff
@@ -46,34 +55,6 @@
       ;; in order to test association of Asset to AssetMappings/AssetProperties
       set-transient-asset-refs))
 
-(deftest bulk-for-asset-related-entities
```

Comment:
is it testing somewhere else now?

Reply:
Yes, a more robust test that doesn't use implementation details was written for https://github.com/threatgrid/ctia/pull/1343